src/apps/chifra/pkg/rpcClient/get_receipt.go

```diff
@@ -106,7 +106,7 @@ func GetTransactionReceipt(chain string, bn uint64, txid uint64, txHash *common.
 	// TODO: are 2 issues with it: 1. circular dependency with types package, 2. every
 	// TODO: call to GetSpecials parses CSV file, so we need to call it once and cache
 	londonBlock := uint64(12965000)
-	if chain == "mainnet" {
+	if tx != nil && chain == "mainnet" {
```

Comment:
Can you please add a TODO here: // TODO: chain specific

Reply:
Added 👍